hip_data_tools/google/sheets.py

```diff
@@ -0,0 +1,82 @@
+import gspread
+
+
+class SheetUtil:
+    """
+    Utility class for connecting to google sheets
+    Args:
+        credentials (ServiceAccountCredentials): credentials to read google sheets
+    """
+
+    def __init__(self, credentials):
+        self.gc = gspread.authorize(credentials)
+
+    def get_value_matrix(self, workbook_name, sheet_name, cell_range='', skip_top_rows_count=0):
+        """
+        Get the values of the sheet as a 2D array
+        Args:
+            workbook_name (string): name of the workbook
+            sheet_name (string): name of the sheet
+            cell_range (string): range of the cells that need to be selected
+            skip_top_rows_count (integer):
+        Returns: values of the sheet as a 2D array
+
+        """
+        worksheet = self.gc.open(workbook_name).worksheet(sheet_name)
+        list_of_lists = []
+
+        if not cell_range:
+            list_of_lists = worksheet.get_all_values()
+            del list_of_lists[0:skip_top_rows_count]
+            return list_of_lists
+        else:
+            for cell in worksheet.range(cell_range):
+                list_of_lists.append(cell.value)
+            return list_of_lists
+
+    def get_table_settings(self, table_name, field_names, s3_bucket, s3_dir):
+        """
+        Get the table settings dictionary
+        Args:
+            table_name (string): name of the athena table
+            field_names (list): list of field names
+            s3_bucket (string): s3 bucket name
+            s3_dir (string): s3 directory
+        Returns: table settings dictionary
+
+        """
+        table_settings = {
+            "table": table_name,
+            "exists": True,
+            "partitions": [],
+            "columns": [],
+            "storage_format_selector": "parquet",
+            "s3_bucket": s3_bucket,
+            "s3_dir": s3_dir,
+            "encryption": False
+        }
+        columns = []
+        for field_name in field_names:
+            columns.append({"column": field_name, "type": "string"})
+        table_settings["columns"] = columns
+
+        return table_settings
+
+    def get_the_insert_query(self, table_name, values_matrix):
```

Comment:
should not exist in SheetUtil

Reply:
Moved this method to google_sheet_to_athena etl class